assets/styl/_modules/form-widget.twig

```diff
@@ -0,0 +1,26 @@
+<form xmlns="http://www.w3.org/1999/html">
+    <div class="clearfix">
+        <div class="form-field-widget col-md-4 pb2">
+            <p>
+                <label for="text_11">Text :</label>
+                <br /><input type="text" id="text_11" aria-describedby="text_11_desc" />
+            </p>
+            <p id="text_11_desc" class="form-help">Field description/help text</p>
+        </div>
+        <div class="form-field-widget col-md-4 pb2">
+            <p>
+                <label for="text_12">Required text <span class="form-required">*</span> :</label>
```

Comment:
in english there do not use space before `:`
in french use non-breakable space before `:`